pkg/bindings/common.go

```diff
@@ -0,0 +1,99 @@
+package bindings
+
+import (
+	"fmt"
+	"math/big"
+
+	evmcappb "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/chain-capabilities/evm"
+	chain_common "github.com/smartcontractkit/chainlink-common/pkg/loop/chain-common"
+	evmtypes "github.com/smartcontractkit/chainlink-common/pkg/types/chains/evm"
```

Comment:
I don't know if I would make the generated code to depend chainlink-common. If we do then users will be sort of aware of chainlink-common and things like versioning of chainlink-common, etc. Can we avoid it?

Reply:
Hm, I'm not sure how we can avoid it in this case since the [QueryTrackedLogsRequest](https://github.com/smartcontractkit/chainlink-common/blob/142aef37e6a562044f671a776430334559c90617/pkg/chains/evm/evm.pb.go#L2106-L2113) is dependent on these types.